test/Garnet.test/RespBlockingListTests.cs

```diff
@@ -231,5 +231,25 @@ public void BlockingClientEventsTests(string blockingCmd)
             ClassicAssert.IsTrue(blockingTask.IsCompletedSuccessfully);
             ClassicAssert.IsTrue(releasingTask.IsCompletedSuccessfully);
         }
+
+        [Test]
+        public void BasicBlockingListPopPushTest()
```

Comment:
This doesn't really test the blocking functionality, you can piggyback on the BasicBlockingListMoveTest and just add a test case for BRPOPLPUSH.

Reply:
Done